addons/website/models/website.py

```diff
@@ -1449,16 +1449,20 @@ def _disable_unused_snippets_assets(self):
 
         for snippet_module, snippet_id, asset_version, asset_type, _ in snippets_assets:
             is_snippet_used = self._is_snippet_used(snippet_module, snippet_id, asset_version, asset_type, html_fields)
-            filename_type = 'scss' if asset_type == 'css' else asset_type
-            assets_path = f'{snippet_id}/{asset_version}.{filename_type}'
+            if asset_type == 'css':
+                asset_path = f'{snippet_id}/{asset_version}.scss'
+                asset_variables_path = f'{snippet_id}/{asset_version}_variables.scss'
+                assets_regex = f'({asset_path}|{asset_variables_path})'
+            else:
+                assets_regex = f'{snippet_id}/{asset_version}.{asset_type}'
```

Comment:
Maybe we could do more generic in one line compared to the previous two lines:
```python
assets_regex = f'{snippet_id}/{asset_version}.+{asset_type)'
```
?
(not tested, just an idea)

Reply:
It's working and it's better :+1:, I am adding a small comment, I don't really know if it is necessary (I can remove it)
By the way by testing I realized that the regex used at https://github.com/odoo/odoo/commit/fa1c05e7bb2959c6025a806a4348f69ed225b4ec#diff-5e92e473fa4d3da6db7ef727fb217dad51ef6c2383913edca73fe040a23e82c2R1408 was returning only the first match of the snippet in the html field... meaning that if a view had a snippet up to date and then an outdated snippet of the same type, the old assets were not enabled. I changed it by adding the 'g' flag. I am adding that to the PR and commit message

Reply:
Nice catch, so you can probably also quickly test it by just inserting an update snippet in the outdated page in the tour, before the outdated snippet, right?

Reply:
Awesome, especially the `Without the flag, it breaks the test`, sounds like nothing and obvious but sadly not how people think those days, thanks for having tested it properly and mentioned it :+1: